pyformat.py

```diff
@@ -160,6 +171,11 @@ def parse_args(argv):
                         help='drill down directories recursively')
     parser.add_argument('-a', '--aggressive', action='count', default=0,
                         help='use more aggressive formatters')
+    parser.add_argument('--remove-all-unused-imports', action='store_true',
+                        help='remove all unused imports, not just standard library \
+(requires "aggresive")')
+    parser.add_argument('--remove-unused-variables', action='store_true',
```

Comment:
I think it would be good to add a check to make sure that `--aggressive` is used. Otherwise, the user might wonder why it isn't working. Maybe something like:

```python
if args.remove_unused_variables and not args.aggressive:
    parser.error('--remove-unused-variables requires --aggressive')
```